r/R/feather.R

```diff
@@ -154,7 +154,10 @@ read_feather <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
     vars_select(names(reader), !!col_select)
   }
 
-  out <- reader$Read(columns)
+  out <- tryCatch(
+    reader$Read(columns),
+    error = function(e) { read_compressed_error(e) }
```

Comment:
I don't see a `read_compressed_error()` function in this PR, and since the tests seem to have passed, that means that none of the regular CI jobs exercises this code path. So in addition to supplying the missing code, you should think about which of our nightly test jobs would exercise this condition. Indeed, you might want to run that job on this PR now, before you fix the missing code, just to demonstrate that it is failing now. That way when you run it after fixing your code, we'll see that the issue has been fixed because the test passes.